privacy-policy.md

```
@@ -0,0 +1,179 @@
+- Allowing you to avoid entering your Personal Access Token repeatedly
+
+---
+
+## Data Sharing and Disclosure
+
+We do not sell, share, or disclose your data.
+
+This is because OrgExplorer does not collect your data in the first place.
+
+The only external communication occurs directly between:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)